### PR TITLE
WinLibrary addon

### PR DESCRIPTION
**Pull request by ELePors**
* Adding WinLibrary feature + tests 
used to manage external DLL in Pharo

### CODE REVIEW

File: src/OS-Windows-Environment-Tests/WinLibraryTest.class.st [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Class {
+	#name : #WinLibraryTest,
+	#superclass : #TestCase,
+	#category : #'OS-Windows-Environment-Tests-Processing'
+}
+
+{ #category : #running }
+WinLibraryTest >> setUp [
+
+	super setUp.
```
Comment (astares) [inline src/OS-Windows-Environment-Tests/WinLibraryTest.class.st:13]: Why an empty #setUp with just a super call? Not necessary. Same for #tearDown

**Comment by ELePors** [comment]
You are right, you can remove it… J

**Comment by ELePors** [comment]
Can you ignore these two methods or do you want me to recreate a Pull Request ? (i can directly update my branch ? or do i have to cancel the pull request and recreate it )

**Comment by astares** [comment]
@ELePors Just delete the two methods from your branch. The PR will update and I can merge then. 

If this is too complicated for you tell me and I will merge and fix afterwards.

**Comment by ELePors** [comment]
It's done !

**Comment by astares** [comment]
@ELePors Thank you!

**Pull request merged**